Geometry/Records/interface/CaloTowerGeometryRecord.h

```diff
@@ -27,6 +28,7 @@ class CaloTowerGeometryRecord :
                 IdealGeometryRecord,
 		CaloTowerAlignmentRcd, 
 		CaloTowerAlignmentErrorRcd,
```

Comment:
@ASvyatkovskiy - I think, you should remove AlignmentErrorRcd from here then. Otherwise, we'll have to keep it in a GT.